From d6e57bfa4f62702e4959bac14126c1e1e454ec0b Mon Sep 17 00:00:00 2001
From: Hubert Chao <hchao@chromium.org>
Date: Tue, 7 Feb 2023 10:14:41 +0000
Subject: [PATCH] Disable CCV/CRS on cronet; its not currently used and it
 makes the library bigger.

Bug: b/267749933
Change-Id: I8a8921891c34fffb17ab467f78748e9ada48001e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4224873
Reviewed-by: David Benjamin <davidben@chromium.org>
Commit-Queue: Stefano Duo <stefanoduo@google.com>
Reviewed-by: Stefano Duo <stefanoduo@google.com>
Cr-Commit-Position: refs/heads/main@{#1102061}
Test: none
---
 net/features.gni | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/net/features.gni b/net/features.gni
index 7441fa2799ab6..6b0bf25cae16e 100644
--- a/net/features.gni
+++ b/net/features.gni
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//build/config/cronet/config.gni")
 import("//build/config/features.gni")
 
 declare_args() {
@@ -37,7 +38,8 @@ declare_args() {
 
   # Platforms where the cert verifier comparison trial is supported.
   # See https://crbug.com/649026.
-  trial_comparison_cert_verifier_supported = is_mac || is_win || is_android
+  trial_comparison_cert_verifier_supported =
+      is_mac || is_win || (is_android && !is_cronet_build)
 
   # Platforms for which certificate verification can be performed using the
   # builtin cert verifier with the Chrome Root Store, and this can be
@@ -45,5 +47,6 @@ declare_args() {
   # flag is false, verification may be done with the platform verifier or the
   # builtin verifier using platform roots, depending on the platform.
   # See https://crbug.com/1216547 for status.
-  chrome_root_store_supported = is_win || is_mac || is_android
+  chrome_root_store_supported =
+      is_win || is_mac || (is_android && !is_cronet_build)
 }
-- 
2.40.0.348.gf938b09366-goog

